static/js/settings_org.js

```diff
@@ -384,26 +390,41 @@ function _set_up() {
 
             success: function (response_data) {
                 $alerts.hide();
+                var success_msg;
                 if (response_data.allow_message_editing !== undefined) {
                     // We expect message_content_edit_limit_seconds was sent in the
                     // response as well
                     var data_message_content_edit_limit_minutes =
                         Math.ceil(response_data.message_content_edit_limit_seconds / 60);
                     if (response_data.allow_message_editing) {
                         if (response_data.message_content_edit_limit_seconds > 0) {
+                            success_msg = "Users can now edit topics for all their messages, and the content of messages which are less than __num_minutes__ minutes old.";
+                            if (response_data.edit_topics_by_admins_only) {
+                                success_msg += " Only admins can edit the topic of any message.";
+                            } else {
+                                success_msg += " Users can edit the topic of any message.";
+                            }
                             ui_report.success(
-                                i18n.t("Users can now edit topics for all their messages, and the content of messages which are less than __num_minutes__ minutes old.",
-                                       {num_minutes : data_message_content_edit_limit_minutes}),
+                                i18n.t(success_msg,
```

Comment:
This won't work for i18n -- strings to be translated have to be string literals.

Reply:
iirc we're deleting all these success strings anyway, though maybe confirm with tabbott.

Reply:
We plan to simplify the strings story for our settings, but no work has been done and it's not a small project, so we're stuck with doing something reasonable here for now.